services/QuillLMS/client/app/bundles/Teacher/components/classrooms/clever_classrooms_empty_modal.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react'
+const emptyClassSrc = `${process.env.CDN_URL}/images/illustrations/empty-class.svg`
+
+interface CleverClassroomsEmptyModalProps {
+  close: () => void;
+}
+
+export default class CleverClassroomsEmptyModal extends React.Component<CleverClassroomsEmptyModalProps, {}> {
```

Comment:
You don't need to change it, but this is a prime example of a component that would be really easy to write as a functional component, which is what we're trying to move to for all React code ultimately. Random example of a functional component here: https://github.com/empirical-org/Empirical-Core/blob/feat/bolding-word-diff/services/QuillLMS/client/app/bundles/Teacher/components/dashboard/welcome_modal.tsx#L14.